src/platforms/java/usage.mdx

```diff
@@ -308,25 +399,69 @@ public class MyClass {
 }
 ```
 
+```kotlin
+import io.sentry.Sentry
+import io.sentry.SentryEvent
+import io.sentry.SentryLevel
+import io.sentry.protocol.Message
+
+
+class MyClass {
+
+    fun main(args: Array<String>) {
+        // NOTE: Replace the test DSN below with YOUR OWN DSN
+        Sentry.init { it.dsn = "https://f7f320d5c3a54709be7b28e0f2ca7081@sentry.io/1808954" }
+    }
+
+    fun unsafeMethod() {
+        throw UnsupportedOperationException("You shouldn't call this!")
+    }
+
+    fun logSimpleMessage() {
+        // This sends an event to Sentry.
+        val event = SentryEvent()
+        val message = Message()
+        message.message = "This is a test"
+        event.message = message
+        event.level = SentryLevel.INFO
+        event.logger = MyClass::class.java.name
+        Sentry.captureEvent(event)
+    }
+
+    fun logException() {
+        try {
+            unsafeMethod()
+        } catch (e: Exception) {
+            // This sends an exception event to Sentry.
+            val event = SentryEvent()
+            val message = Message()
+            message.message = "Exception caught"
+            event.message = message
+            event.level = SentryLevel.INFO
+            event.logger = MyClass::class.java.name
+            event.throwable = e
+            Sentry.captureEvent(event)
```

Comment:
what's about using the kotlin `apply()` function? so we avoid all the `event.` or `message.`